SupportFiles/ws-tools.sh

```diff
@@ -0,0 +1,161 @@
+
+# Configure VNC server
+cp /lib/systemd/system/vncserver@.service  /etc/systemd/system/vncserver@:1.service
+sed -i 's/<USER>/WorkstationUser/g' /etc/systemd/system/vncserver@:1.service
+systemctl daemon-reload
+systemctl start vncserver@:1
+systemctl enable vncserver@:1
+
+# Add firewall VNC server firewall rules
+setenforce 0
+firewall-cmd --add-port=5901/tcp
+firewall-cmd --add-port=5901/tcp --permanent
+setenforce 1
+
+# Unzip the tools.tar.gz
+tar xfz /etc/cfn/tools/tools.tar.gz -C /etc/cfn/tools || err_exit "Failed to unzip tools.tar.gz"
+
+# Install Anaconda
+bash /etc/cfn/tools/anaconda/anaconda.sh -b -p /home/WorkstationUser/anaconda3
+chown -R WorkstationUser:WorkstationUser /home/WorkstationUser/anaconda3
+
+cp /etc/cfn/tools/anaconda/anaconda.desktop /usr/share/applications/anaconda.desktop
+cp /etc/cfn/tools/anaconda/anaconda.desktop /home/WorkstationUser/.local/share/applications/anaconda.desktop
+chown WorkstationUser:WorkstationUser /home/WorkstationUser/.local/share/applications/anaconda.desktop
+chmod 600 /home/WorkstationUser/.local/share/applications/anaconda.desktop
+
+# Install ATOM
+# Launch it from your terminal by running the command "atom"
+yum -y install /etc/cfn/tools/atom/atom.x86_64.rpm || err_exit "Failed to install ATOM"
+
+#Install Eclipse NEON IDE for Java EE Developers
+tar xfz /etc/cfn/tools/eclipse/eclipse-jee-neon-3-linux-gtk-x86_64.tar.gz -C /opt/ || err_exit "Failed to unzip eclipse-jee-neon-3-linux-gtk-x86_64.tar.gz"
+ln -s /opt/eclipse/eclipse /usr/local/bin/eclipse
+
+cp /etc/cfn/tools/eclipse/eclipse.desktop /usr/share/applications/eclipse.desktop
+cp /etc/cfn/tools/eclipse/eclipse.desktop /home/WorkstationUser/.local/share/applications/eclipse.desktop
+chown WorkstationUser:WorkstationUser /home/WorkstationUser/.local/share/applications/eclipse.desktop
+chmod 600 /home/WorkstationUser/.local/share/applications/eclipse.desktop
+
+#Install Intellij
+tar xfz /etc/cfn/tools/intellij/ideaIC-2018.3.3.tar.gz -C /opt/ || err_exit "Failed to unzip ideaIC-2018.3.3.tar.gz"
+chmod -R 755 /opt/idea-IC-183.5153.38
+ln -s /opt/idea-IC-183.5153.38/bin/idea.sh /usr/local/bin/idea
+
+cp /etc/cfn/tools/intellij/jetbrains-idea-ce.desktop /usr/share/applications/jetbrains-idea-ce.desktop
+cp /etc/cfn/tools/intellij/jetbrains-idea-ce.desktop /home/WorkstationUser/.local/share/applications/jetbrains-idea-ce.desktop
+chown WorkstationUser:WorkstationUser /home/WorkstationUser/.local/share/applications/jetbrains-idea-ce.desktop
+chmod 600 /home/WorkstationUser/.local/share/applications/jetbrains-idea-ce.desktop
+
+#Install emacs
+yum -y install emacs || err_exit "Failed to install emacs"
+
+#Install Gradle
+unzip -d /opt/ /etc/cfn/tools/gradle/gradle-5.1.1-bin.zip
+chmod -R 755 /opt/gradle-5.1.1
+ln -s /opt/gradle-5.1.1/bin/gradle /usr/local/bin/gradle
+
+# Install Maven
+yum -y install maven || err_exit "Failed to install maven"
+
+# Install git
+rpm -Uvh /etc/cfn/tools/git/endpoint-repo-1.7-1.x86_64.rpm
```

Comment:
Really, really, _really_ bad practice to execute `rpm` commands for install or remove of RPMs on `yum`-managed systems. Replace with an appropriate `yum` command

Reply:
Not done. Need help on this.

Reply:
Done. Check in the changes already.